dracut/30ignition/ignition-setup.sh

```diff
@@ -1,22 +1,30 @@
 #!/bin/bash
+set -eux
 
-set -e
+bootmnt=/mnt/boot_partition
 
-# OEM directory in the initramfs itself
-src=/usr/share/oem
+# Grab our ignition configs from:
+#  - A platform specific directory for this platform
+#  - The boot partition (user/installer overrides)
+sources=("/usr/share/platforms/${OEM_ID}/"
```

Comment:
It may make sense for this part to move into Ignition itself: Ignition could look for `/usr/lib/ignition/platform/${OEM_ID}/base.ign` and fall back to `/usr/lib/ignition/base.ign` if missing.

Reply:
Agreed, but this is fine for now IMO